Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -0,0 +1,95 @@
+
+import Foundation
+import MapboxDirections
+import SwiftCLI
+
+
+private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
+
+class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > : Command {
+    var name = "process"
+    
+    @Key("-i", "--input", description: "Filepath to the input JSON.")
```

Comment:
Relatedly, #476 requests the ability to convert a Directions API request URL to RouteOptions, and #375 tracks using GPX as an input format. Neither enhancement is required for this PR, but one of them may be desirable as tail work. Creating the input JSON file requires looking at the source code of this library to know which keys are expected; these keys aren’t standardized as part of the Directions API.

Reply:
If #375 and #476 aren’t feasible right now, another possibility would be to add the available options as command-line flags, such as `--profileIdentifier` and `--allowsUTurnAtWaypoint`, as suggested in https://github.com/mapbox/mapbox-directions-swift/issues/451#issue-688324859, but that might be difficult to keep in sync with the model objects.

Reply:
Sorry, could not find that suggestion in linked comment. Is it a correct link?

Reply:
we made it so you have to pass a filepath to the input json to android cli too. that will be good for directions api, to make the request once and then verify it on both libraries.

had a follow up ticket for handling Directions API requests https://github.com/mapbox/navigation-sdks/issues/639
I think I like the approach from #476, where we input a directions api url and parse it.

but we should also support the "Filepath to the input JSON"

Reply:
I should’ve said it more explicitly, but this bit:

> RouteOptions or MatchOptions properties would need to be specified as command line options and flags, since those properties aren’t part of the API response format.

was supposed to imply flags such as `--profileIdentifier` and `--allowsUTurnAtWaypoint`. As I mentioned above, it isn’t critical for this PR but would be nice to have in the long run to make this tool something we commonly use in our day-to-day workflows.

> but we should also support the "Filepath to the input JSON"

Yes, I’m glad this PR implements the option to provide the input in JSON format for automated workflows. However, I think some other format would also be desirable, since the property names of any JSON input object would probably end up differing between platforms. We could agree on a standard for such input files, which would complement the existing Directions API documentation, but it wouldn’t be obvious or particularly convenient for manual workflows.